content.js

```diff
@@ -6,7 +6,6 @@ function compareRight(s1, s2) {
 var isWikiwand = compareRight(document.location.hostname, 'wikiwand.com');
 
 function replaceUnbalancedBraces(text) {
-  console.log('Balancing ' +  text);
```

Comment:
Won't it make sense to remove other log messages as well? E.g. "Replacing PNG images with MathJax..."